gravitee-plugin-annotation-processors/src/test/java/io/gravitee/plugin/annotation/processor/result/TestConfiguration.java

```diff
@@ -29,6 +29,8 @@
 @ConfigurationEvaluator(attributePrefix = "gravitee.attributes.endpoint.test")
 public class TestConfiguration {
 
+    private static final String CONSTANT = "my_constant_should_be_ignored";
```

Comment:
Is there any assertion to check after adding this constant in order to validate the expected behavior?

Reply:
Without the modification made to check the modifiers in the annotation processor, adding this throws an error at compile time.